docs/explanation/backend-tables.md

```diff
@@ -28,7 +28,7 @@ Make the core tables available for use in a dataset definition
 with import statements like:
 
 ```python
-from ehrql.tables.beta.core import medications, patients
+--8<-- 'includes/code/tutorial/writing-a-dataset-definition/asthma_medications-standalone-success/analysis/dataset_definition.py:tables_import'
```

Comment:
comment: I probably should document briefly how the snippet support works, though it's deducible from the changes here.

Essentially:

* for including the whole file, just specify the file path
* for including part of the file, you'll need to use `# --8<-- [start:snippet_id]` and `# --8<-- [end:snippet:id]
* if you also want to include a whole file which contains snippet markers, you'll need to create a snippet that wraps the whole file content so that you can use those markers (nested snippet markers are removed)